MH4U Dex/ItemSourcesTableViewController.m

```diff
@@ -0,0 +1,112 @@
+//
+//  ItemSourcesTableViewController.m
+//  MH4U Dex
+//
+//  Created by Joseph Goldberg on 2/19/15.
+//  Copyright (c) 2015 Joseph Goldberg. All rights reserved.
+//
+
+#import "ItemSourcesTableViewController.h"
+
+#import <CoreData/CoreData.h>
+
+#import "Constants.h"
+#import "CoreDataController.h"
+
+#import "MonsterContainerViewController.h"
+#import "RegionContainerViewController.h"
+
+#import "Area.h"
+#import "AreaDrop.h"
+#import "Drop.h"
+#import "Item.h"
+#import "Monster.h"
+#import "MonsterDrop.h"
+
+#import "ItemSourceTableViewCell.h"
+
+@interface ItemSourcesTableViewController ()
+
+@property (nonatomic, strong) NSArray *sources;
+
+@end
+
+@implementation ItemSourcesTableViewController
+
+- (void)viewDidLoad
+{
+    [super viewDidLoad];
+    
+    NSManagedObjectContext *managedObjectContext = [CoreDataController sharedCDController].managedObjectContext;
+    NSError *fetchError = nil;
+    NSFetchRequest *itemFetchRequest = [[NSFetchRequest alloc] initWithEntityName:[Item entityName]];
+    itemFetchRequest.predicate = [NSPredicate predicateWithFormat:@"%K == %@", ItemAttributes.name, self.itemName];
+    Item *item = (Item *)[managedObjectContext executeFetchRequest:itemFetchRequest error:&fetchError].firstObject;
+    NSFetchRequest *fetchRequest;
+    if (self.isMonsterSource) {
+        fetchRequest = [[NSFetchRequest alloc] initWithEntityName:[MonsterDrop entityName]];
+        fetchRequest.predicate = [NSPredicate predicateWithFormat:@"%K == %@", MonsterDropRelationships.item, item];
+        fetchRequest.sortDescriptors = @[[NSSortDescriptor sortDescriptorWithKey:MonsterDropAttributes.id ascending:YES]];
+        self.tableView.accessibilityIdentifier = MHDItemMonsterSources;
+    } else {
+        
+        fetchRequest = [[NSFetchRequest alloc] initWithEntityName:[AreaDrop entityName]];
+        fetchRequest.predicate = [NSPredicate predicateWithFormat:@"%K == %@", AreaDropRelationships.item, item];
+        fetchRequest.sortDescriptors = @[[NSSortDescriptor sortDescriptorWithKey:AreaDropAttributes.idDecimalString ascending:YES]];
```

Comment:
You might be right!  I need to convert it from a Set to an Array, though, and I want to keep the sort descriptor.  I'll look into it.